src/metatrain/experimental/nanopet/model.py

```diff
@@ -413,7 +409,46 @@ def forward(
 
         atomic_features_dict: Dict[str, torch.Tensor] = {}
         for output_name, head in self.heads.items():
-            atomic_features_dict[output_name] = head(node_features)
+            if self.atomic_basis_target_info[output_name][
+                "type"
+            ] == "atomic_basis_spherical" and self.atomic_basis_target_info[
+                output_name
+            ]["sample_kind"].startswith("per_pair"):
```

Comment:
This is not great to read. Maybe you could extract two variable and change this to something like `if atomic_basis_is_spherical and sample_is_per_pair`

Reply:
Sure